src/components/common/MapContainer.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { string } from 'prop-types';
+import GoogleMapReact from 'google-map-react';
+import PositionMarker from 'components/common/PositionMarker';
+
+class MapContainer extends React.Component {
+  static defaultProps = {
+    key: process.env.GOOGLE_MAPS_KEY
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      center: { lat: this.props.lat, lng: this.props.lng },
+      zoom: 16
+    };
+  }
+
+  componentDidMount() {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition((position) => {
```

Comment:
You can destructure `position` object here
```suggestion
      navigator.geolocation.getCurrentPosition(({ coords: { latitude, longitude } }) => {
```